src/main/kotlin/anttracker/Release.kt

```diff
@@ -23,33 +26,115 @@ value class ReleaseId(
             "Release id length must be between 1 and 8 characters"
         }
     }
+
+    override fun toString(): String = id
 }
 
-class Release(
-    val releaseName: ReleaseId,
-    val product: Product,
-    var releaseDate: LocalDate,
-)
+val promptEnterRel = "\nPlease enter new release name. ` to abort:"
+
+val promptSelectRel = "\nPlease select release. ` to abort:"
+
+val promptSelectAffRel = "\nPlease select affected release. ` to abort:"
 
 // -------------------------------------------------------------------------------
 // Prints a sub-menu of selecting a product to create a release for.
 // Use by including in main-menu loop; should be triggered upon user selecting
 // New Release from the main menu. Thus, upon function return, system returns to main menu.
 // ---
 fun menu() {
-    TODO()
+    println("== NEW RELEASE ==")
+    // val selectedProduct = selectProduct()    <- TODO: uncomment and replace when implemented in product module
+    val selectedProduct = "Product 1"
+    when (val selection = readln()) {
+        "`" -> return
+        "1" -> createRelease(selectedProduct)
+        else -> {
+            println("Bad input: $selection.")
```

Comment:
On bad input, return to main menu?

Reply:
I'd say give the user the ability to retry on a misinput, and only abort/go back if some transaction actually fails or user enters a backtick.